### PR TITLE
Fix implicit namespace autocompletion. Resolves: #959

### PR DESCRIPTION
**Pull request by m-novikov**

### CODE REVIEW

**Comment by MohamedAlFahim** [comment]
After reading some of the Travis CI logs for your pull request, I saw that the tests failed to install specific packages in some versions of Python.
Here's part of the message:
> invocation failed (exit code 1), logfile: /home/travis/build/davidhalter/jedi/.tox/py33/log/py33-1.log

**Comment by davidhalter** [comment]
They already have been applied, The test results are just old here.

**Comment by davidhalter** [comment]
Thanks man. I should really have a look at this one. Sorry for not acting sooner. It's just quite a bit of code and sometimes I'm too tired to look at PRs.

File: jedi/_compatibility.py [hunk 1]
```diff
@@ -161,6 +161,58 @@ def find_module_pre_py33(string, path=None, fullname=None):
...
+    # Copy of pkgutil.iter_modules adapted to work with namespaces
+
+    for path in paths:
+        importer = pkgutil.get_importer(path)
+
+        if not isinstance(importer, importlib.machinery.FileFinder):
+            for mod_info in pkgutil.iter_modules([path], prefix):
+                yield mod_info
+            continue
+
```
Comment (davidhalter) [inline jedi/_compatibility.py]: Please comment that every thing from here on can be found in the function `pkgutil._iter_file_finder_modules`.

**Comment by elliott-beach** [comment]
@m-novikov Bump, are you working on this? I think this would resolve https://github.com/Microsoft/vscode-python/issues/231 in the python extension for Visual Studio Code.

**Comment by m-novikov** [comment]
Sorry, had no time. I will do it this week.

**Comment by davidhalter** [comment]
Now the testing structure has changed by quite a bit. Please try to use `Script` as a pytest fixture. You won't need imports anymore.

**Comment by davidhalter** [comment]
I just changed a couple of things. Thanks for the work!

**Pull request merged**